gui/src/api.js

```diff
@@ -32,3 +32,5 @@ export const getJobSpec = (id) => request(`/v2/specs/${id}`)
 export const getJobSpecRuns = (id, page, size) => request(`/v2/specs/${id}/runs`, {page: page, size: size})
 
 export const getAccountBalance = () => request('/v2/account_balance')
+
+export const getConfiguration = () => request('/v2/config')
```

Comment:
As a champion of ubiquitous language, I would be remiss if I didn't point out the discrepancy between our API URL (`/v2/config`) and our gui url (`/configuration`). Let's consolidate the two to the same spelling. Which one should we use? I'm leaning towards the shorter one: `config`.